server.js

```diff
@@ -28,6 +149,61 @@ app.use(express.json());
 app.get("/", (req, res) => {
   res.send("Hello Technigo!");
 });
+const { Schema } = mongoose;
+const userSchema = new Schema ({
+  name: String,
+  age: Number,
+  alive: Boolean
+});
+
+const User = mongoose.model("User", userSchema);
+
+const songSchema = new Schema({
+    id: Number,
+    trackName: String,
+    artistName: String,
+    genre: String,
+    bpm: Number,
+    energy: Number,
+    danceability: Number,
+    loudness: Number,
+    liveness: Number,
+    valence: Number,
+    length: Number,
+    acousticness: Number,
+    speechiness: Number,
+    popularity: Number
+})
+
+const Song = mongoose.model("Song", songSchema);
+
+
+
+app.get("/songs/id/:id", async (req, res) => {
+  try {
+    const singleSong = await Song.findById(req.params.id);
+    if (singleSong) {
+      res.status(200).json({
+        success: true,
+        body: singleSong
+      })
+    } else {
+      res.status(404).json({
+        success: false,
+        body: {
+          message: "Song not found"
+        }
+      })
+    }
+  } catch(e) {
```

Comment:
I like to use 'err' instead of 'e' to make it very clear that it stands for error and not event (since event in JS is often shortened to a single e as well). The context makes it clear what you mean here and it is fully correct, it's just a personal preference of mine to add clarity